R/geom-.r

```diff
@@ -123,6 +123,32 @@ Geom <- ggproto("Geom",
       data[names(missing_eval)] <- missing_eval
     }
 
+    if (length(modifiers) != 0) {
+      env <- new.env(parent = baseenv())
```

Comment:
Since we already rely on rlang, I think using `env()` would make the code simpler here:

```
env <- env(baseenv(), mapped = mapped)
stage_mask <- env(emptyenv(), stage = stage_mapped)
```